embulk-input-jdbc/build.gradle

```diff
@@ -6,3 +6,50 @@ embulkPlugin {
     category = "input"
     type = "jdbc"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Tomohiro Hashidate"
+                        email = "kakyoin.hierophant@gmail.com"
+                    }
```

Comment:
@joker1007 Hi!

We wonder if you're okay to nominate you in `pom.xml` of `embulk-input-jdbc/postgresql` on Maven Central, from your work in #36. I picked up your name and email from the GitHub account, and the Git commit log, which are already public.

Please let me know if you don't want, or if you want another name/email.

Reply:
Thank you so much! It's my honor!!

Name and email are no problem.